README.md

```diff
@@ -10,6 +10,12 @@ Smart-contracts can be found [here](https://github.com/district0x/name-bazaar/tr
 
 ## Starting a dev server with an Ethereum Testnet
 
+First, clone the project with submodules _([ens](https://github.com/ensdomains/ens))_
```

Comment:
What if you have cloned the repository before and want to just load the submodules now, i.e. position I'm in when reviewing now?
I gathered I should use:
`git submodule update --init --recursive`
Please add this case to README as well.